unified-runtime/scripts/core/device.yml

```diff
@@ -464,6 +464,10 @@ etors:
       desc: "[$x_bool_t] support for native bfloat16 conversions"
     - name: KERNEL_LAUNCH_CAPABILITIES
       desc: "[$x_kernel_launch_properties_flags_t] Bitfield of supported kernel launch properties."
+    - name: LUID
+      desc: "[uint8_t[]][optional-query] return device LUID"
```

Comment:
I'd prefer "return device Windows LUID", just to make it clearer this is a Windows thing.

Reply:
Sure. Node mask is also windows specific so I have added that there as well.